pkg/sdn/plugin/controller.go

```diff
@@ -24,35 +24,29 @@ import (
 )
 
 const (
-	// rule versioning; increment each time flow rules change
-	VERSION        = 3
-	VERSION_TABLE  = "table=253"
-	VERSION_ACTION = "actions=note:"
-
 	BR    = "br0"
 	TUN   = "tun0"
 	VXLAN = "vxlan0"
 
 	VXLAN_PORT = "4789"
```

Comment:
VERSION is used in controller.go and ovscontroller.go (const defined in ovscontroller.go)
TUN is used in controller.go and ovscontroller.go (const defined in controller.go)
VXLAN is used in ovscontroller.go (const defined in controller.go)
VXLAN_PORT is used in node_iptables.go (const defined in controller.go)

May be we can place these constants in a separate file or at least in one place so that it is easy to search.